docs/content.md

```diff
@@ -287,6 +288,41 @@ hero_bg_color: white
 hero_blend_content: true
 ```
 
+### Values
+
+You can use the Values system to maintain key values (e.g. salaries, dates, fees etc) in a single file, and then use these values throughout the site's content. Set up a list of values in one or more YML files stored in the `config/values/` folder (or sub-folder), for example `config/values/dates.yml`:
+
+```yaml
+dates:
+  example:
+    opening: 1st September 2024
+
+dates_example_closing: 31/12/2024
+```
+
+These values can then be used in markdown files by referencing the value as `$``value_name``$`, e.g. `$``dates_example_opening``$` or `$``dates_example_closing``$`. Note that structured composite keys will be flattened to a single key.
```

Comment:
@martyn-w curious about the use of single and then double backtick in this part of the docs, it renders like this:

<img width="1020" alt="image" src="https://github.com/DFE-Digital/get-into-teaching-app/assets/35870975/fcf9c73c-6060-4728-960d-dd85755ea010">

I find that quite confusing as I wasn't sure whether to put the backticks in. This might be my poor understanding of markdown, but can we not write the variable without it?

Reply:
I know it's then added in the markdown implementation below, just curious about the disparity.

Reply:
Sorry @MylesJarvis my mistake, I'll fix this